docker-entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "Updating zotero translators"
+git submodule update --recursive --remote
```

Comment:
This will update the Zotero client submodule too, which could have unexpected consequences. We should do a `git pull origin master` in the `modules/translators` submodule instead.

(I'm also [planning](https://github.com/zotero/translation-server-v2/issues/1) to build translator auto-updating into the server code, the same as in the client, but this can do for the moment, and mirrors what we do in production for v1.)